tests/Unleash.Tests/Communication/UnleashHttpClientFactory_RegisterHttpClient_Tests.cs

```diff
@@ -0,0 +1,40 @@
+﻿using NUnit.Framework;
+using System;
+
+namespace Unleash.Tests.Communication
+{
+    public class UnleashHttpClientFactory_RegisterHttpClient_Tests
+    {
+        private Uri _apiUri { get; } = new Uri("http://unleash.herokuapp.com/");
```

Comment:
You can use fields instead of properties, it's more simple and more readable.
Apart form that, the .NET properties name convention is not the same as fields.